packages/cli/test/unit/import-lb3-model/migrate-model.test.js

```diff
@@ -77,6 +77,74 @@ describe('importLb3ModelDefinition', () => {
     });
   });
 
+  context('model properties with db settings', () => {
+    let modelData;
+
+    const STRING_PROPERTY = {
+      type: 'String',
+      required: false,
+      length: 25,
+      precision: null,
+      scale: null,
+      postgresql: {
+        columnName: 'name',
+        dataType: 'character varying',
+        dataLength: 25,
+        dataPrecision: null,
+        dataScale: null,
+        nullable: 'YES',
+      },
+    };
+
+    const NUMERIC_PROPERTY = {
+      type: 'Number',
+      required: false,
+      length: null,
+      precision: 64,
+      scale: 0,
+      postgresql: {
+        columnName: 'count',
+        dataType: 'bigint',
+        dataLength: null,
+        dataPrecision: 64,
+        dataScale: 0,
+        nullable: 'YES',
+      },
+    };
+
+    before(function modelWithCustomDbSettings() {
+      const properties = {
+        name: STRING_PROPERTY,
+        count: NUMERIC_PROPERTY,
+      };
+      const MyModel = givenLb3Model('MyModel', properties, {});
+      modelData = importLb3ModelDefinition(MyModel, log);
+    });
+
+    it('connector metadata is migrated for string property', () => {
+      expect(modelData.properties)
+        .to.have.property('name')
+        .deepEqual({
+          type: `'string'`,
+          tsType: 'string',
+          length: 25,
+          postgresql: `{columnName: 'name', dataType: 'character varying', dataLength: 25, dataPrecision: null, dataScale: null, nullable: 'YES'}`,
```

Comment:
@deepakrkris I'm just wondering why a string and not an object passed here. Why is this not like this?:
```typescript
{
  type: `'string'`,
  tsType: 'string',
  length: 25,
  postgresql: {
    columnName: 'name',
    dataType: 'character varying',
    dataLength: 25,
    dataPrecision: null,
    dataScale: null,
    nullable: 'YES'
  },
},
```

Reply:
@derdeka thanks for the review. The unit test is for the function `importLb3ModelDefinition` it returns json values as strings. But the template generator takes care of converting into json objects.